common/dna_data/wc_orc_dna.txt

```diff
@@ -0,0 +1,1195 @@
+ 		gene_bs_nose_size={ "nose_size_neg" 8 "nose_size_neg" 8 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 15 "nose_tip_angle_pos" 197 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_neg" 255 "nose_tip_forward_neg" 193 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 125 "nose_tip_width_neg" 89 }
+ 		face_detail_cheek_def={ "cheek_def_01" 255 "cheek_def_02" 175 }
+ 		face_detail_cheek_fat={ "cheek_fat_01_pos" 49 "cheek_fat_01_neg" 6 }
+ 		face_detail_chin_cleft={ "chin_dimple" 89 "chin_dimple" 16 }
+ 		face_detail_chin_def={ "chin_def" 255 "chin_def" 13 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 115 "eye_lower_lid_def" 243 }
+ 		face_detail_eye_socket={ "eye_socket_03" 253 "eye_socket_03" 137 }
+ 		face_detail_nasolabial={ "nasolabial_01" 255 "nasolabial_03" 255 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_neg" 255 "nose_ridge_def_neg" 0 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 0 "nose_tip_def" 164 }
+ 		face_detail_temple_def={ "temple_def" 255 "temple_def" 161 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_03" 174 "brow_wrinkles_01" 235 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_01" 240 "eye_wrinkles_01" 240 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_03" 255 "forehead_wrinkles_01" 191 }
+ 		expression_other={ "cheek_wrinkles_both_01" 245 "cheek_wrinkles_both_01" 245 }
+ 		complexion={ "complexion_2" 255 "complexion_5" 219 }
+ 		gene_height={ "normal_height" 168 "normal_height" 163 }
+ 		gene_bs_body_type={ "body_fat_head_fat_medium" 155 "body_fat_head_fat_low" 155 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 255 "body_shape_pear_half" 217 }
+ 		gene_bs_bust={ "bust_clothes" 90 "bust_shape_2_full" 90 }
+ 		gene_age={ "old_3" 255 "old_3" 87 }
+ 		gene_eyebrows_shape={ "avg_spacing_lower_thickness" 0 "avg_spacing_avg_thickness" 117 }
+ 		gene_eyebrows_fullness={ "layer_2_lower_thickness" 0 "layer_2_lower_thickness" 132 }
+ 		gene_body_hair={ "body_hair_avg" 120 "body_hair_sparse" 126 }
+ 		hairstyles={ "no_hairstyles" 0 "orcish_hairstyles" 24 }
+ 		beards={ "indian_beards" 255 "orcish_beards" 77 }
+ 		eye_accessory={ "normal_eyes" 46 "normal_eyes" 46 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "no_eyelashes" 222 "no_eyelashes" 222 }
+ 		gene_bs_ear_lenght={ "erect_ear_lenght" 30 "erect_ear_lenght" 29 }
+ 		special_eyebrows={ "no_eyebrows" 111 "no_eyebrows" 111 }
+ 		special_eyes={ "magic_eyes" 255 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_tusks_pos" 255 "mouth_tusks_pos" 255 }
+ 		static_tusks={ "orcish_tusks" 127 "orcish_tusks" 193 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+ 		gene_scars_battle={ "scars_battle" 255 "scars_battle" 0 }
+ 		special_headgear_eye_patch={ "eye_patch" 127 "eye_patch" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+orgrim_doomhammer_dna = { # by Garthor13
```

Comment:
He has a very weak chin compared to other orcs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114061047-08d3f100-98a7-11eb-9ccc-26c4f17f3385.png)

</details>

Reply:
Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.